### PR TITLE
Simplify latency alerting

### PR DESCRIPTION
**Pull request by yuriatgoogle**
Fixes #

## Proposed Changes

* loosened default latency SLO target
* added latency-alert module
* added per-service toggles for latency alerts
* added per-service toggles for latency SLOs

**Release Note**

```release-note
Latency alerting done via threshold, rather than SLO
```

### CODE REVIEW

File: terraform/alerting/module.latency-alert/main.tf [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Google LLC
```
Comment (sethvargo) [inline terraform/alerting/module.latency-alert/main.tf:1]: This is 2021 now for new files

File: terraform/alerting/module.latency-alert/variables.tf [hunk 2]
```diff
@@ -0,0 +1,32 @@
+variable "custom_service_id" {
```
Comment (sethvargo) [inline terraform/alerting/module.latency-alert/variables.tf:1]: Need copyright header

File: terraform/alerting/slos.tf [hunk 3]
```diff
@@ -71,11 +94,12 @@ module "latency-slos" {
 
   project = var.project
 
-  enabled               = var.https-forwarding-rule != ""
+  # enabled               = var.https-forwarding-rule != ""
```
Comment (mariliamelo) [inline terraform/alerting/slos.tf:97]: remove this line?

File: terraform/alerting/slos.tf [hunk 4]
```diff
@@ -14,27 +14,36 @@
...
+    enable_latency_alert   = false
+    latency_alert_duration = 300000 # 5 minutes, in ms
+
   }
   service_configs = {
     adminapi = merge(local.default_per_service_slo,
-      { enable_alert = true,
+      { enable_alert         = true,
+        enable_latency_alert = true,
+        enable_latency_slo   = false,
```
Comment (mariliamelo) [inline terraform/alerting/slos.tf:30]: It seems you don't need to set this here, since it is false by default?

**Review by sethvargo — APPROVED** [review]
/lgtm

File: terraform/alerting/module.latency-alert/main.tf (hunk 1 above)
Comment (yuriatgoogle) [reply terraform/alerting/module.latency-alert/main.tf]: Done, thank you.

File: terraform/alerting/module.latency-alert/variables.tf (hunk 2 above)
Comment (yuriatgoogle) [reply terraform/alerting/module.latency-alert/variables.tf:15]: Added, thank you.

File: terraform/alerting/module.latency-alert/variables.tf [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+variable "threshold" {
+  type        = number
+  description = "Latency threshold (in ms)."
+}
+
+variable "duration" {
+  type        = number
+  description = "Duration of alert evaluation"
```
[the remark this answers is not in the document]
Comment (yuriatgoogle) [reply terraform/alerting/module.latency-alert/variables.tf]: Added unit, thank you.

File: terraform/alerting/slos.tf (hunk 3 above)
Comment (yuriatgoogle) [reply terraform/alerting/slos.tf]: Good catch, thank you.

File: terraform/alerting/slos.tf (hunk 4 above)
Comment (yuriatgoogle) [reply terraform/alerting/slos.tf]: Yes, thank you.

**Pull request merged**